src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -343,6 +359,32 @@ contract WorldIDIdentityManagerImplV1 is OwnableUpgradeable, UUPSUpgradeable, IW
         return _latestRoot;
     }
 
+    function sendRootToStateBridge() public virtual onlyProxy onlyInitialized {
+        if(stateBridgeProxy == address(0x0)) revert stateBridgeHasBeenTurnedOff();        
```

Comment:
As mentioned above, we don't want to revert here. Just return. Please use the brace-style of `if` even for single statements.

Reply:
many production codebases use this pattern though

Reply:
I fixed the revert with a check in `registerIdentities`

Reply:
I will use brace ifs, but style-wise there is nothing wrong with using inline ifs in Solidity for simple things.